src/ast/mod.rs

```diff
@@ -1430,6 +1430,19 @@ pub enum Statement {
         // Specifies the actions to perform when values match or do not match.
         clauses: Vec<MergeClause>,
     },
+    /// CACHE TABLE <table_name> [[AS] [QUERY]]
```

Comment:
What database is this meant to follow? It doesn't seem to follow Spark's syntax

https://docs.databricks.com/spark/latest/spark-sql/language-manual/sql-ref-syntax-aux-cache-cache-table.html

How is it different than a `CREATE VIEW` statement?

Reply:
Thanks for the doc, I will support the full Cache table syntax.

The difference between `CACHE TABLE` and `CREATE VIEW` is that users can cache small tables in memory before Join to speed up queries, such as dimension tables in data warehouse.The View should not be able to do this if it is not a memory table.